### PR TITLE
Meme Generator

### PR DESCRIPTION
**Pull request by Sambhav250703**
Mini project on Meme Generator using HTML, CSS & Javascript for Hacktoberfest 2022.

### CODE REVIEW

**Comment by Pranavtechie** [comment]
Hey @Sambhav250703, can you convert your project into a guided project. 

Also can you rename the file from `spec.yaml` to `spec.yml` for consistency

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
- converting the project into a guided project 
- updating the file name to `spec.yml`

**Comment by Sambhav250703** [comment]
Guided project means I must explain all the steps right how I developed this project and which api I had used ?

**Comment by Sambhav250703** [comment]
Please give a update and label my pr for hacktoberfest.

**Comment by Sambhav250703** [comment]
https://github.com/codedamn/projects/issues/40

**Comment by Sambhav250703** [comment]
Can you check and merge the pr so it will count for my hacktoberfest 2022

File: Meme Generator/spec.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    In this project, you're supposed to build a meme generator page. This includes the following features:
+
+    - Generate meme
+    - Display author name
+    - Loading
+    - meme display
+
+    A detailed breakdown is available in the README.md file once you start the project. All the best!
+    
+host-files: https://sambhav250703.github.io/Hacktoberfest-codedamn/
```
Comment (Pranavtechie) [inline Meme Generator/spec.yml:18]: remove host-files

**Review by Pranavtechie — DISMISSED** [review]

File: Meme Generator/spec.yml (hunk 1 above)
Comment (Sambhav250703) [reply Meme Generator/spec.yml]: Done changes pls check it out and merge the pr

**Comment by Sambhav250703** [comment]
Any more changes I want to do for get accepted and merging the pr @Pranavtechie

**Review by Pranavtechie — APPROVED** [review]

**Pull request merged**